pkgs/tools/graphics/stegsolve/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchurl, jre, makeWrapper, copyDesktopItems, makeDesktopItem }:
+
+stdenv.mkDerivation rec {
+  pname = "stegsolve";
+  version = "1.3";
+
+  src = fetchurl {
+    # No versioned binary is published :(
```

Comment:
can we maybe get one from some other place, like archive.org?

Reply:
There is only a single published version as far as I can tell 🤷‍♀️

Reply:
`1.3` sort of implies otherwise, but if it hasn't changed for a long time, maybe it's fine?

idk if we really have a policy on this kind of thing. even if the link contains a version, it can just be republished and I've personally seen this on more than one occasion, in the past.

Reply:
The earliest version on archive.org is from 2013 and is also `1.3` https://web.archive.org/web/20130104120022/http://www.caesum.com:80/handbook/Stegsolve.jar